src/System.Security.Cryptography.OpenSsl/tests/EcDsaOpenSslTests.cs

```diff
@@ -252,14 +252,16 @@ private static void Exercise(this ECDsaOpenSsl e)
         private const int NID_secp384r1 = 715;
         private const int NID_secp521r1 = 716;
 
-        [DllImport("libcrypto")]
-        private static extern IntPtr EC_KEY_new_by_curve_name(int nid);
+        private const string CryptoNative = "System.Security.Cryptography.Native";
```

Comment:
This is in a test assembly, and I didn't think we had InternalsVisibleTo the test assemblies.  (Libraries is internal).  Or do you think I should be compiling in the Libraries.cs into the test assembly?